lib/board.ex

```diff
@@ -4,25 +4,21 @@ defmodule Board do
     0..last_board_space |> Enum.to_list
```

Comment:
Really like the transition to `game` as the first argument in here.  This module, `Board`, does a lot of operations on `Game`.  Maybe it's a submodule?  `Game.Board`?  That could be a good step forward to an interface refactor.